tuxemon/states/persistance/load_menu.py

```diff
@@ -32,8 +32,7 @@ def on_menu_selection(self, menuitem: Optional[MenuItem[None]]) -> None:
                 self.client.pop_state(old_world)
             except ValueError:
                 # when game is loaded from the start menu
-                self.client.pop_state()  # close this menu
```

Comment:
why are these removed?

Reply:
> why are these removed?

The game seemed to work without those functions when I added the new solution. There wasn't any difference with having them removed, so I'll add them back right now.